src/Workspaces/Core/Portable/Rename/RenameLocation.ReferenceProcessing.cs

```diff
@@ -327,7 +327,8 @@ private static string TrimNameToAfterLastDot(string name)
                                 if (location.IsInSource)
                                 {
                                     var token = location.FindToken(cancellationToken);
-                                    if (!syntaxFacts.IsKeyword(token) && token.ValueText == referencedSymbol.Name)
+                                    if (!syntaxFacts.IsReservedOrContextualKeyword(token) && 
+                                        token.ValueText == referencedSymbol.Name)
```

Comment:
note: i'm not sure the above is correct logic.  but i'm not changing things here.  jsut trying to preserve exiting logic.